notebooks/introduction_to_tensorflow/solutions/3_keras_sequential_api_vertex_preprocessing_layers.ipynb

```diff
@@ -0,0 +1,584 @@
+{
```

Comment:
<p>* Maybe we can use <code>model.save</code> instead </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/195/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>